index.bs

```diff
@@ -4211,6 +4304,8 @@ an [=attribution trigger=] |trigger|:
     :: |trigger|'s [=attribution trigger/aggregatable source registration time configuration=].
     : [=aggregatable attribution report/trigger context ID=]
     :: |trigger|'s [=attribution trigger/trigger context ID=]
+    : [=aggregatable attribution report/filtering ID max bytes=]
+    :: |trigger|'s [=attribution trigger/aggregatable filtering ID max bytes=]
```

Comment:
Should we update this for null reports as well?

Reply:
good catch, updated